schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/conf/SchemaRegistryTestConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2017 Hortonworks.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package com.hortonworks.registries.schemaregistry.avro.conf;
+
+public class SchemaRegistryTestConfiguration {
+    private String serverYAMLPath;
+    private String clientYAMLPath;
+    private final String v1APIPath = "api/v1";
```

Comment:
This is a constant and it can be moved to `SchemaRegistryClientUtil` where this is used.